client/my-sites/email/email-management/email-home.tsx

```diff
@@ -88,9 +89,9 @@ const EmailHome = ( props: EmailManagementHomeProps ) => {
 		selectedIntervalLength,
 		sectionHeaderLabel,
 		source,
-	} = props;
+	}: EmailManagementHomeProps = props;
 
-	const selectedSite = useSelector( getSelectedSite );
+	const selectedSite: SiteData | null = useSelector( getSelectedSite );
```

Comment:
This type definition is actually also redundant – Typescript can infer the appropriate type by looking at the return type of the `getSelectedSite` function (which in this case is also `SiteData | null`)